src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md

```diff
@@ -140,21 +148,26 @@ Frequently used elements in your extension should use mobile-optimised [Material
+Rebuild your extension’s layout using the [Material Web Components Layout Grid](https://material-components.github.io/material-components-web-catalog/#/component/layout-grid).
 
 ### Components
 
-Most if not all elements in your extension should use mobile-optimised [Material Web Components](https://material-components.github.io/material-components-web-catalog/):
+Most, if not all, elements in your extension should use mobile-optimized [Material Web Components](https://material-components.github.io/material-components-web-catalog/):
 - [Tab bar](https://material-components.github.io/material-components-web-catalog/#/component/tabs)
 - [Top app bar](https://material-components.github.io/material-components-web-catalog/#/component/top-app-bar)
 
-Seldom used elements that could be mobile-optimised include [Card](https://material-components.github.io/material-components-web-catalog/#/component/card), [Chips](https://material-components.github.io/material-components-web-catalog/#/component/chips), [Data table](https://material-components.github.io/material-components-web-catalog/#/component/data-table), [Dialog](https://material-components.github.io/material-components-web-catalog/#/component/dialog), [Drawer](https://material-components.github.io/material-components-web-catalog/#/component/drawer), [Image list](https://material-components.github.io/material-components-web-catalog/#/component/image-list), and [Snackbar](https://material-components.github.io/material-components-web-catalog/#/component/snackbar).
+Seldom used elements that could be mobile-optimized include [Card](https://material-components.github.io/material-components-web-catalog/#/component/card), [Chips](https://material-components.github.io/material-components-web-catalog/#/component/chips), [Data table](https://material-components.github.io/material-components-web-catalog/#/component/data-table), [Dialog](https://material-components.github.io/material-components-web-catalog/#/component/dialog), [Drawer](https://material-components.github.io/material-components-web-catalog/#/component/drawer), [Image list](https://material-components.github.io/material-components-web-catalog/#/component/image-list), and [Snackbar](https://material-components.github.io/material-components-web-catalog/#/component/snackbar).
+
+<!--
+Components should match [Firefox for Android color usage](https://www.figma.com/proto/TebIgESfWmQkMcEmwNaYZl/Android?node-id=0%3A498&viewport=-4823%2C213%2C0.5&scaling=min-zoom&hide-ui=1) for light and dark themes. 
+
+This link requires figma login, any idea where it should go?
 
-Components should match [Firefox for Android color usage](https://www.figma.com/proto/TebIgESfWmQkMcEmwNaYZl/Android?node-id=0%3A498&viewport=-4823%2C213%2C0.5&scaling=min-zoom&hide-ui=1) for light and dark themes.
+-->
```

Comment:
Probably best to delete this part too.

Reply:
Done – created https://github.com/mozilla/extension-workshop/issues/1659 to cover.